examples/overview.ipynb

```diff
@@ -0,0 +1,806 @@
+    "\n",
+    "Above, we looked at which residues were more commonly in contact with GTP201. But what if we wanted to know which atom-atom contacts involved GTP201? We can do that with `contacts.most_common_atoms_for_residue`. "
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 17,
+   "metadata": {},
+   "outputs": [
+    {
+     "data": {
+      "text/plain": [
+       "[([ASP119-CG, GTP201-N1], 0.6138613861386139),\n",
+       " ([GTP201-C2, ASP119-CG], 0.6138613861386139),\n",
+       " ([GTP201-C2, ASP119-OD2], 0.6138613861386139),\n",
+       " ([GTP201-N2, ASP119-CG], 0.6138613861386139),\n",
+       " ([ASP119-OD2, GTP201-N1], 0.6138613861386139),\n",
+       " ([GTP201-N2, ASP119-OD2], 0.6138613861386139),\n",
+       " ([GTP201-N2, LEU120-CD2], 0.6039603960396039),\n",
+       " ([ASP119-OD1, GTP201-N1], 0.6039603960396039),\n",
+       " ([GTP201-C2, ASP119-OD1], 0.6039603960396039),\n",
+       " ([GTP201-N2, ASP119-OD1], 0.6039603960396039)]"
+      ]
+     },
+     "execution_count": 17,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "traj_contacts.most_common_atoms_for_residue(gtp201)[:10]"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "On the other hand, what if you wanted to know the most common atoms for a specific contact pair? Let's do that for the contact pair Val81-Asn116. First we'll get the residue objects, then we'll use the `most_common_atoms_for_contact` method."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 18,
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "VAL81 ASN116\n"
+     ]
+    }
+   ],
+   "source": [
+    "val81 = topology.residue(80)\n",
+    "asn116 = topology.residue(115)\n",
+    "print(val81, asn116)"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 19,
+   "metadata": {},
+   "outputs": [
+    {
+     "data": {
+      "text/plain": [
+       "[([ASN116-CB, VAL81-CG1], 0.9702970297029703),\n",
+       " ([VAL81-CG1, ASN116-CG], 0.24752475247524752),\n",
+       " ([VAL81-CG1, ASN116-ND2], 0.21782178217821782),\n",
+       " ([VAL81-CG1, ASN116-N], 0.0594059405940594)]"
+      ]
+     },
+     "execution_count": 19,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "traj_contacts.most_common_atoms_for_contact([val81, asn116])"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 20,
+   "metadata": {},
+   "outputs": [
+    {
+     "data": {
+      "text/plain": [
+       "1809"
+      ]
+     },
+     "execution_count": 20,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "traj_contacts.most_common_atoms_for_contact([val81, asn116])[0][0][0].index"
+   ]
+  },
```

Comment:
Is this leftover debug code, or was this to indicate how to get the atom index for asn116-cb?
```suggestion
```
or add some description about what is done here